src/main/webapp/WEB-INF/views/car/addCar.jsp

```diff
@@ -0,0 +1,19 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add car</title>
+</head>
+<body>
+<h1>Please the form below</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Manufacturer <select name="manufacturer_id">
+                    <c:forEach items="${manufacturers}" var="manufacturer">
+                        <option value="${manufacturer.id}">${manufacturer.name}</option>
+                    </c:forEach>
+</select><br>
+    Model <input type="text" name="model"><br>
```

Comment:
```suggestion
    Model <input type="text" name="model" required><br>
```

Reply:
So we can to avoid creating car with empty model. Same with Driver an Manufacturer